bootstrap/utils/utils.go

```diff
@@ -161,3 +167,47 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+func LoadFile(path string, contents *any) error {
+	var fileBytes []byte
+	var err error
+
+	lowerPath := strings.ToLower(path)
+
+	if strings.Contains(lowerPath, "http") {
+		resp, err := http.Get(lowerPath)
```

Comment:
Will ned to use http.Request and request.Do in order to add the auth header and set an explicit timeout.

Reply:
how will we get the header and timeout information?

Reply:
header is info is from the secret data from the SecretProvider as described in the ADR.
The timeout can be the `Service.RequestTimeout` from config except when loading the config file.
Best to add secretProvider and requestTimeout parameters to this function and leave it to callers to find the values to pass in..